src/intTest/resources/fhir/pathology.fhir.json

```diff
@@ -127,22 +127,84 @@
           "coding": [
             {
               "system": "http://loinc.org",
-              "code": "42R4.",
-              "display": "Serum ferritin"
+              "display": "B12/FOLATE"
```

Comment:
Is it ok to set the system when there's no code?

Reply:
Not sure... Documentation isn't clear about it, but I agree it's suspicious. As it stands, we're kind of saying "if there is a code, it will be according to this system, but there isn't lulz yolo"

Okay, maybe not the last bits.

![image](https://user-images.githubusercontent.com/38751994/110456568-a037ff80-80c1-11eb-95ac-7c2cc03bb1c5.png)
 
Just says they're all optional - nothing about being dependent on other fields.

Reply:
I agree the documentation is not clear at all. If you check out message 29 in our UAT Pathology test data ticket https://gpitbjss.atlassian.net/browse/NIAD-961, we didn't set the `system` when the `code` was not present. So you could do this?